### PR TITLE
dev-python/llvmlite: Add version 0.16

### PR DESCRIPTION
**Pull request by jvesely**
compatible with newly stable llvm-3.9

### CODE REVIEW

File: dev-python/llvmlite/llvmlite-0.16.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+"
+DEPEND="${RDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]
+"
+PATCHES=(
+	"${FILESDIR}"/llvmlite-0.15.0-use-system-six.patch
+)
+
+python_prepare_all() {
+	sed -e 's/-flto$/-flto -fPIC/' \
```
Comment (mgorny) [inline dev-python/llvmlite/llvmlite-0.16.0.ebuild]: Side note: we really oughtn't allow build systems to force `-flto`.

**Comment by mgorny** [comment]
@jvesely, do the tests pass for you? I'm on 4.0 already, so it's hard for me to test it.

**Comment by mgorny** [comment]
@jvesely, I've pushed an additional flag fix to your branch. Could you test if it works for you, please?

**Comment by jvesely** [comment]
THe mofied ebuild fails to apply the sed command:
```
sed: -e expression #2, char 23: missing command
 * ERROR: dev-python/llvmlite-0.16.0::gentoo failed (prepare phase):
 *   (no error message)
 * 
 * Call stack:
 *     ebuild.sh, line  115:  Called src_prepare
 *   environment, line 2641:  Called distutils-r1_src_prepare
 *   environment, line  804:  Called python_prepare_all
 *   environment, line 2462:  Called die
 * The specific snippet of code:
 *       sed -e 's/-static-libstdc++/-fPIC/' -e '/d^(CXX|LD)_FLTO_FLAGS/' -i ffi/Makefile.linux || die;
```
the following change fixes the problem:
```
-               -e '/d^(CXX|LD)_FLTO_FLAGS/' \
+               -e '/^(CXX|LD)_FLTO_FLAGS/d' \
```

and my simple test case runs OK.

**Comment by mgorny** [comment]
Yeah, thanks. I must've accidentally hit some key combo in vim ;-).

**Pull request closed without merging**